reviewer-checklist.md

```diff
@@ -0,0 +1,66 @@
+
+<H2>Reviewers, please use the following checklists as guidance for your review:</H2>
+
+<H2>Overall Impression of the Submission</H2>
+
+Please give the submitter and other reviewers your overall impression of the dataset and supplementary materials
+
+
+
+<H2>Data Organisation</H2>
+
+The dataset is organized according to the following principles:
+
+
+- [ ] The data is "Tidy" (if appropriate to the data type)
```

Comment:
Perhaps we can include a link to a resource for the reviewer to learn more about what this means. I think we have one such link already in the README?